src/openrct2-ui/windows/Sign.cpp

```diff
@@ -560,20 +589,22 @@ static void window_sign_small_dropdown(rct_window *w, rct_widgetindex widgetInde
  *
  *  rct2: 0x006E60D5
  */
-static void window_sign_small_invalidate(rct_window *w)
+static void window_sign_small_invalidate(rct_window * w)
 {
-    rct_widget* main_colour_btn = &window_sign_widgets[WIDX_MAIN_COLOUR];
-    rct_widget* text_colour_btn = &window_sign_widgets[WIDX_TEXT_COLOUR];
+    rct_widget * main_colour_btn = &window_sign_widgets[WIDX_MAIN_COLOUR];
```

Comment:
We switched to left-aligning in loco. Might be worth considering?

Reply:
Not without team discussion. Until that time, the project standard should be applied as-is.

Reply:
Probably best to stick to the current standard and change the whole code base at once, if we decide to change it, yes.